domain-cc/cc-app/src/python_src/api.py

```diff
@@ -67,6 +66,16 @@ def get_classification(claim: Claim) -> Optional[PredictedClassification]:
         else:
             logging.info("No dropdown match for contention_text")
 
+    if claim.claim_type == "new":
```

Comment:
what is the difference between having `claim.claim_type == "new"` and `if claim.contention_text` from above?

Is what we're doing applicable to all new claims and should we try to handle them w/ the same logic?